bundles/org.openhab.binding.broadlinkthermostat/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="broadlinkthermostat" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Broadlinkthermostat Binding</name>
+	<description>This is the binding for Broadlinkthermostat devices.</description>
+	<author>Florian Mueller</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.

Reply:
Removed.